fbpic/fields/fields.py

```diff
@@ -446,7 +532,7 @@ def partial_interp2spect(self, fieldtype) :
         ---------
         fieldtype :
             A string which represents the kind of field to transform
-            (either 'E', 'B', 'J', 'rho_next', 'rho_prev')
+            (either 'E', 'B', 'J', 'rho_next', 'rho_prev’, ‘A’)
```

Comment:
Could you replace the back ticks here (`) by regular ticks (')?
I think that this is what is making the Python2 tests fail.